Resources/Prototypes/Reagents/Consumable/Drink/soda.yml

```diff
@@ -165,6 +165,13 @@
         damage:
           types:
             Poison: 1
+      - !type:HealthChange
+        conditions:
+        - !type:OrganType
+          type: Slime
+        damage:
+          types:
+            Poison: -1
```

Comment:
just add an inverse organ condition to the poison damage effect instead of balancing it

Reply:
changed